vega_sim/api/market.py

```diff
@@ -111,6 +112,7 @@ class MarketConfig(Config):
             "lp_price_range": 0.5,
             "linear_slippage_factor": 1e-3,
             "quadratic_slippage_factor": 0,
+            "successor": "default",
```

Comment:
And then maybe default to `None` here?
```suggestion
            "successor": None,
```